Print_A_Bot/controls/templates/base.html

```diff
@@ -1,12 +1,17 @@
 <!DOCTYPE html>
 
+{% load static %}
 
 <html>
 
 <head>
 	
 	<title>{% block title %}{% endblock title %}</title>
 
+  <!--[if lt IE 9]><script src="{% static 'js/html5shiv.js' %}"></script><![endif]-->
```

Comment:
this and the block below loads up bootstrap and some of the fancy js features that the djwifi templates require to work